zcash_client_backend/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+### Changed
+- Bumped dependencies to `bls12_381 0.8`, `group 0.13`, 
+- Bumped dev dependency to `jubjub 0.10`
```

Comment:
We don't need to document changes to dev dependencies; they aren't observable outside the crate.
```suggestion
```